docs/001_develop/01_development-environment/002_launchpad/index.mdx

```diff
@@ -440,6 +432,10 @@ Then as soon as you start to type Genesis, the available commands are displayed
 | `Stop processes`      | Kills all running Genesis processes on the JVM or server and the WebUi. Use this command when processes are unresponsive. |
 | `Unpack project`      | Unpacks an available project generated in Genesis Create, and makes it active in the workspace. |
 
```

Comment:
Add documentation for Genesis Sync command, I will let you discover what it does.